test-data/drugbank_drugs.smi

```diff
@@ -0,0 +1 @@
+../lib/galaxy/datatypes/test/drugbank_drugs.smi
```

Comment:
@nsoranzo are links a good idea here, or would you prefer to transfer them to https://github.com/galaxyproject/galaxy-test-data as well?

Reply:
Links seem to cause problems. I will finish this in a week.

Reply:
Maybe move these files to test-data/ and symlink them back to `lib/galaxy/datatypes/test/` , that seems to be the current pattern.

Reply:
Done. Lets see if it works